server/mustache-templates/navPartial.mst

```diff
@@ -3,7 +3,7 @@
     <button class="navbar-toggler navbar-dark" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
         <span class="navbar-toggler-icon"></span>
     </button>
-
+    <!-- Delilah needs more treats -->
```

Comment:
Very cool and very important feature. Thank you for making these changes. Approved!

Reply:
I agree